node/sc/request_manager.go

```diff
@@ -42,3 +43,15 @@ func MakeTransactOpts(accountKey *ecdsa.PrivateKey, nonce *big.Int, chainID *big
 	}
 	return auth
 }
+
+func MakeTransactOptsWithKeystore(wallet accounts.Wallet, from common.Address, nonce *big.Int, chainID *big.Int, gasPrice *big.Int) *bind.TransactOpts {
```

Comment:
How about moving this function to the bridge accounts and deleting this `request_manager.go`?